api/v1alpha1/constants.go

```diff
@@ -32,3 +32,6 @@ const PreviousEnvironmentCommitStatusKey = "promoter-previous-environment"
 
 // ReconcileAtAnnotation is the annotation used to indicate when the webhook triggered a reconcile
 const ReconcileAtAnnotation = "promoter.argoproj.io/reconcile-at"
+
+// CommitStatusDeAggregationAnnotation is the label used to identify commit statuses that make up the aggregated active commit status
+const CommitStatusDeAggregationAnnotation = "promoter.argoproj.io/commit-status-de-aggregation"
```

Comment:
```suggestion
const CommitStatusDeAggregationAnnotation = "promoter.argoproj.io/previous-environment-statuses"
```

Maybe something like that?